### PR TITLE
<Input/> - add border prop

### PR DESCRIPTION
**Pull request by EgoziE**

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6496

**Review by milkyfruit — APPROVED** [review]

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Few questions and comments I was not sure about

File: src/Input/Input.js [hunk 1]
```diff
@@ -394,19 +402,6 @@ Input.defaultProps = {
   hideStatusSuffix: false,
 };
 
-const borderRadiusValidator = (props, propName) => {
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply src/Input/Input.js:397]: It was supposed to be deleted in wsr 7 -> 8 and we missed it by mistake

File: src/TimeInput/TimeInput.js [hunk 2]
```diff
@@ -266,6 +266,7 @@ export default class TimeInput extends Component {
       status,
       hideStatusSuffix,
       statusMessage,
+      border,
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply src/TimeInput/TimeInput.js:269]: Noted!
All non-input components will do in my 3rd PR, thanks

File: src/Input/Input.js [hunk 3]
```diff
@@ -386,6 +393,7 @@ Input.displayName = 'Input';
 Input.defaultProps = {
   autoSelect: true,
   size: 'medium',
+  border: 'default',
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply src/Input/Input.js:396]: You're right, my mistake, should keep one name only

File: src/Input/test/Input.visual.js [hunk 4]
```diff
@@ -85,9 +91,11 @@ export const runTests = (
     return testWithTheme(
       <div>
         <WrappedInputAffix />
-        <WrappedInputAffix roundInput />
         <WrappedInputIconAffix />
-        <WrappedInputIconAffix roundInput />
+        <WrappedInputAffix border="round" />
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply src/Input/test/Input.visual.js:95]: It's the theme's owner to style it.
No action from our side needed.

File: src/Input/index.d.ts [hunk 5]
```diff
@@ -22,6 +22,7 @@ export interface InputProps {
   clearButton?: boolean;
   className?: string;
   name?: string;
+  border?: 'standard' | 'round' | 'bottomLine';
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply src/Input/index.d.ts:25]: will fix thanks!

File: src/Input/test/Input.visual.js [hunk 6]
```diff
@@ -75,6 +75,12 @@ export const runTests = (
             <WrappedInput key={status} status={status} />
             <WrappedInput key={status} status={status} forceHover />
             <WrappedInput key={status} status={status} forceFocus />
+            <WrappedInput
```
[the remark this answers is not in the document]
Comment (EgoziE) [reply src/Input/test/Input.visual.js:78]: No need for double tests.
I'll improve that section in my 2nd or 3rd PR
But thanks!

File: src/Input/Input.js [hunk 7]
```diff
@@ -468,8 +463,8 @@ Input.propTypes = {
   /** Used to reference element data when a form is submitted. */
   name: PropTypes.string,
 
-  /** Borders type */
-  borders: PropTypes.oneOf(['standard', 'round', 'borderless']),
+  /** border type */
+  border: PropTypes.oneOf(['standard', 'round', 'borderless']),
```
Comment (TalZa1810) [inline src/Input/Input.js:467]: `borderless` should be changed to `bottomLine`, no?

Comment (EgoziE) [reply src/Input/Input.js:467]: changing

**Pull request merged**